### ISSUE TITLE
Feature request: Bulk add Instructor training progress

### ISSUE THREAD

**Issue by maneesha** [carpentries/amy#1565]
**Training**
People get listed as learners before they are checked off as having completed the training.  This is because they may have registered for the event but then failed to complete the two day session.  We should be able to check off learners associated with the TTT event and mark "Training" complete for all of them at once.

**Discussion and Demo**
People often complete the discussion and demo session in  groups.  We should have the ability to bulk add them  as having passed or failed the discussion and demo portion of the checkout process.

**Comment by pbanaszkiewicz** [comment]
Hello @maneesha. 

Here: https://amy.carpentries.org/trainings/trainees/?search=&is_instructor=&training=2606&training_request=unknown&order_by=
you have an "Add Progress to Selected Trainees" form (just below the table). By selecting specific trainee items, you can then bulk-add progress to them (either Training or SWC/DC/LC Demo or Discussion). Does this solve your issue?

**Comment by maneesha** [comment]
This solves the issue when it comes to recording the **Training Event** progress, since that is directly associated with one event.

For the Demo and Discussion, we will want to add many people at once, who are not co-associated with any event.  This means that there's no way to select them.  Being able to add their progress will require a new feature.

**Comment by pbanaszkiewicz** [comment]
@maneesha I tested adding `Discussion` to multiple persons. It worked without specifying the related event. I may be missing something, but so far it looks like it solves the issue.

Is your issue that the people are not yet in AMY database when they finish Discussion/Demo parts?

**Comment by maneesha** [comment]
That is correct - we do not need to specify the related event.  But if I have 20 people I want to check off as having completed the **Discussion** who are all coming from different events, I would have to search for each of those 20 people one by one.  I'd like to have a feature like `Bulk Add Persons` which lets us add many people's training progress at once by uploading a CSV file.

**Comment by maneesha** [comment]
This was part of a pre-pandemic project to restructure and streamline the instructor training checkout process. We are tabling this until we discuss more internally.
